.semaphore/semaphore.yml

```diff
@@ -63,11 +63,6 @@ blocks:
           # E2E_K8S_VERSION_OPTION in tests/conftest.py; it should point to the most recent version used in CI.
           - env_var: K8S_VERSION
             values:
-              - v1.15.12
```

Comment:
Can we continue testing the `v1.15.12`? We still have this version in our K8s environment 😄

Reply:
We can remove it once we move out of these clusters 😄

Reply:
I'll put it back in. 👍 Please let us know when when you move on.